app/src/main/java/fr/free/nrw/commons/theme/NavigationBaseActivity.java

```diff
@@ -71,7 +71,7 @@ public void initDrawer() {
         Menu nav_Menu = navigationView.getMenu();
         View headerLayout = navigationView.getHeaderView(0);
         ImageView userIcon = headerLayout.findViewById(R.id.user_icon);
-        if (prefs.getBoolean("login_skipped", true)) {
+        if (prefs.getBoolean("login_skipped", false)) {
```

Comment:
Does "login_skipped=false" mean that login has been skipped?

Reply:
login_skipped ==true means login skipped.